containers/Dashboard/Dashboard.js

```diff
@@ -4,12 +4,12 @@ import HeaderNav from '../../components/HeaderNav/HeaderNav';
 import ShortEventCardContainer from '../../components/ShortEventCardContainer/ShortEventCardContainer';
 import { StyleSheet, View } from 'react-native';
 
-export default function Dashboard() {
+export default function Dashboard({ navigation }) {
   return (
     <View>
-      <HeaderNav />
-      <Map />
-      <ShortEventCardContainer />
+      <HeaderNav navigation={navigation}/>
+      <Map navigation={navigation}/>
+      <ShortEventCardContainer navigation={navigation}/>
```

Comment:
Excellent work tracing the { navigation } path and spotting where it needed to be passed down as a prop. We will prob need to do that go most components moving forward.

Reply:
Agreed! I think we have like 90% of all the components we need but yeah, they'll all need the navigation i believe